source/arm11.s

```diff
@@ -88,7 +94,7 @@ wait_arm11_loop:
 	BX              R1
 
 	pa_hijack_arm9_dst:  .long 0x1FFFFC00
-	pa_arm11_code:       .long 0x1FFFFFF8
```

Comment:
Here is what i was talking about.
This address has nothing to do with the brahma 1 compatibility, since this is the pointer where the arm11 entrypoint will be written to by arm9.
Keeping this to 0x1FFFFFFC (as original) makes the arm11 kernel load the original firm partition after the firmlaunch, which causes any cfw loader to run an updated arm9 kernel with an outdated arm11 one in the otherside.
So, leaving it 0x1FFFFFF8 should not affect in any way Brahma1 compatibility, while improves stability if you wish to run custom code on the arm11.
Can youy please change this in another pull request?